lib/types/star.js

```diff
@@ -227,15 +227,29 @@ class Star extends PrinterType {
 
 
   // ----------------------------------------------------- PRINT IMAGE -----------------------------------------------------
-  async printImage(image) {
+  printImage(imagePath) {
```

Comment:
Although you could definitely argue that it's out of scope for this PR, I'd recommend taking the time to reimplement these image parsing sections asynchronously so that they aren't blocking.

In an Electron context, in particular, this will be blocking the main thread which could cause some significant issues in applications consuming this.

[pngjs Async API](https://github.com/lukeapage/pngjs#async-api)

Seeing as `jpeg-js` doesn't seem to have an async implementation I would also recommend considering migrating to something much more battle-tested that also has more features, such as [JIMP](https://github.com/oliver-moran/jimp). This may allow for the adoption of more features, more easily in the future:

[JIMP](https://github.com/oliver-moran/jimp)